### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by piesieomane**
In this PULL REQUEST the following list below was implemented;

- -- Understand the different types of relationships between tables.
- Prepare complex queries that answer analytical questions.
- Use primary key & foreign key mechanisms for joining tables.

- Create vets, visits, and a specialization table
image

- Insert data into the vets' table
image

-Insert data into the specialization table
image

-Insert data into the visits table
image

-1st query 
image

-2nd query
image

-last query
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -43,4 +43,23 @@ ALTER TABLE animals ADD owners_id BIGINT REFERENCES owners(id);
...
\ No newline at end of file
+REFERENCES owners (parent_key_columns);
+
+--DAY 4 DB
+CREATE TABLE vets (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(200) NOT NULL,
+    age INT,
+    date_of_graduation DATE NOT NULL
+);
```
Comment (mateo951) [inline schema.sql:54]: New table with the name vets ✔

File: schema.sql [hunk 2]
```diff
@@ -43,4 +43,23 @@ ALTER TABLE animals ADD owners_id BIGINT REFERENCES owners(id);
...
+
+CREATE TABLE specialization (
+    vets_id BIGINT REFERENCES vets (id),
+    species_id BIGINT REFERENCES species (id)
+);
```
Comment (mateo951) [inline schema.sql:59]: New join table with the name `specializations` ✔

File: schema.sql [hunk 3]
```diff
@@ -43,4 +43,23 @@ ALTER TABLE animals ADD owners_id BIGINT REFERENCES owners(id);
...
+
+CREATE TABLE visits (
+    vets_id BIGINT REFERENCES vets (id),
+    animals_id BIGINT REFERENCES animals (id),
+    date_of_visit DATE NOT NULL
+);
```
Comment (mateo951) [inline schema.sql:65]: new join table with the name `visits` ✔ Awesome!

File: data.sql [hunk 4]
```diff
@@ -43,3 +43,41 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
...
+	('6', '2', date '2021-04-07'),
+	('7', '3', date '2019-09-29'),
+	('8', '4', date '2020-10-03'),
+	('8', '4', date '2020-11-04'),
+	('9', '2', date '2019-01-24'),
+	('9', '2', date '2019-05-15'),
+	('9', '2', date '2020-02-27'),
+	('9', '2', date '2020-08-03'),
+	('10', '3', date '2020-05-24'),
+	('10', '1', date '2021-01-11');
```
Comment (mateo951) [inline data.sql:83]: Propper data was introduced into the new tables 😁

**Review by mateo951 — APPROVED** [review]
### Hi @piesieomane, 

**🔥 Highlights🔥**
- All queries were awnsered 😎

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 4 above)
Comment (piesieomane) [reply data.sql:83]: Thank you very much

**Pull request merged**